Apps/Common/src/Delegates/AuthServer/IUserDelegate.cs

```diff
@@ -0,0 +1,52 @@
+//-------------------------------------------------------------------------
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Common.Delegates.AuthServer
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using HealthGateway.Common.Models.AuthServer;
+
+    /// <summary>
+    /// The UserDelegate IAM Admin Service interface.
+    /// </summary>
+    public interface IUserDelegate
```

Comment:
Maybe this should be IIdentityManagementDelegate

Reply:
I wanted ot have a class align to each admin topic in the style that Keycloak does. this topic is user management.  I.e. there might be RealmDelegate, ClientDelegate, IdentityProviderDelegate etc.. to admin each of those areas... rather than one big master class that does it all.